Minisign causes a panic

### aqua info

aqua v2.45.0 linux/amd64


### Overview

`aqua i` causes a panic.

### How to reproduce

aqua.yaml

```yaml
packages:
  - name: zigtools/zls@0.14.0
```

Other related code such as local Registry

```
# yaml-language-server: $schema=https://raw.githubusercontent.com/aquaproj/aqua/main/json-schema/registry.json
packages:
  - type: github_release
    repo_owner: zigtools
    repo_name: zls
    description: A Zig language server supporting Zig developers with features like autocomplete and goto definition
    version_constraint: "false"
    version_overrides:
      - version_constraint: "true"
        asset: zls-{{.OS}}-{{.Arch}}-{{.Version}}.{{.Format}}
        format: tar.xz
        windows_arm_emulation: true
        minisign:
          type: github_release
          asset: "{{.Asset}}.minisig"
          public_key: "RWR+9B91GBZ0zOjh6Lr17+zKf5BoSuFvrx2xSeDE57uIYvnKBGmMjOex"
        replacements:
          amd64: x86_64
          arm64: aarch64
          darwin: macos
        overrides:
          - goos: windows
            format: zip
```

Executed command and output

```console
$ aqua i
```


### Expected behaviour

The panic doesn't occur.

### Actual behaviour

```
INFO[0000] download and unarchive the package            aqua_version=2.45.0 env=linux/amd64 package_name=zigtools/zls package_version=0.14.0 program=aqua registry=standard
INFO[0000] verify a package with minisign                aqua_version=2.45.0 env=linux/amd64 package_name=zigtools/zls package_version=0.14.0 program=aqua registry=standard
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x74a728]

goroutine 18 [running]:
github.com/aquaproj/aqua/v2/pkg/minisign.(*ExecutorImpl).exec(0x0, {0xb99bf0?, 0x400041e3c0?}, {0x4000387418?, 0x2004000010002?, 0xd0001?})
        /home/runner/work/aqua/aqua/pkg/minisign/exec.go:73 +0x28
github.com/aquaproj/aqua/v2/pkg/minisign.(*ExecutorImpl).Verify(0x0, {0xb99bf0, 0x400041e3c0}, 0x4000223c00, 0x20?, {0x400000ec80?, 0x5?})
        /home/runner/work/aqua/aqua/pkg/minisign/exec.go:90 +0xe8
github.com/aquaproj/aqua/v2/pkg/minisign.(*Verifier).Verify(0x40004217a0, {0xb99bf0, 0x400041e3c0}, 0x4000223c00, 0x400000ebe0?, 0xb?, 0x2?, 0x4000060708?, 0x400046e2e0)
        /home/runner/work/aqua/aqua/pkg/minisign/verifier.go:41 +0xc8
github.com/aquaproj/aqua/v2/pkg/installpackage.(*minisignVerifier).Verify(0x400041f100, {0xb99bf0, 0x400041e3c0}, 0x4000223c00, {0x400000ec40, 0xe})
        /home/runner/work/aqua/aqua/pkg/installpackage/verify_minisign.go:51 +0x208
github.com/aquaproj/aqua/v2/pkg/installpackage.(*Installer).download(0x400036b540, {0xb99bf0, 0x400041e3c0}, 0x4000223b20, 0x4000387cf8)
        /home/runner/work/aqua/aqua/pkg/installpackage/download.go:155 +0x9b4
github.com/aquaproj/aqua/v2/pkg/installpackage.(*Installer).downloadWithRetry(0x400036b540, {0xb99bf0, 0x400041e3c0}, 0x4000223ab0, 0x4000387cf8)
        /home/runner/work/aqua/aqua/pkg/installpackage/download.go:28 +0x290
github.com/aquaproj/aqua/v2/pkg/installpackage.(*Installer).InstallPackage(0x400036b540, {0xb99bf0, 0x400041e3c0}, 0x4000223a40, 0x4000387ef8)
        /home/runner/work/aqua/aqua/pkg/installpackage/installer.go:274 +0x32c
github.com/aquaproj/aqua/v2/pkg/installpackage.(*Installer).InstallPackages.func1()
        /home/runner/work/aqua/aqua/pkg/installpackage/installer.go:232 +0x8c
golang.org/x/sync/errgroup.(*Group).Go.func1()
        /home/runner/go/pkg/mod/golang.org/x/sync@v0.11.0/errgroup/errgroup.go:78 +0x54
created by golang.org/x/sync/errgroup.(*Group).Go in goroutine 1
        /home/runner/go/pkg/mod/golang.org/x/sync@v0.11.0/errgroup/errgroup.go:75 +0x98
```

### Note

https://aquaproj.github.io/docs/reference/registry-config/minisign/

https://github.com/aquaproj/aqua/blob/5ac9556eda91457a7a46c8ba9a5f46e61883f77b/pkg/minisign/exec.go#L73

--- a/.github/workflows/wc-integration-test.yaml
+++ b/.github/workflows/wc-integration-test.yaml
@@ -312,6 +312,7 @@ jobs:
         env:
           - runs-on: windows-latest
           - runs-on: ubuntu-24.04
+          - runs-on: ubuntu-24.04-arm
           - runs-on: macos-13
           - runs-on: macos-14
     env:

--- a/pkg/minisign/exec.go
+++ b/pkg/minisign/exec.go
@@ -70,6 +70,12 @@ func wait(ctx context.Context, logE *logrus.Entry, retryCount int) error {
 }
 
 func (e *ExecutorImpl) exec(ctx context.Context, args []string) error {
+	if e == nil {
+		return errors.New("executor is nil")
+	}
+	if e.executor == nil {
+		return errors.New("e.executor is nil")
+	}
 	_, err := e.executor.ExecStderr(osexec.Command(ctx, e.minisignExePath, args...))
 	return err //nolint:wrapcheck
 }
@@ -77,6 +83,9 @@ func (e *ExecutorImpl) exec(ctx context.Context, args []string) error {
 var errVerify = errors.New("verify with minisign")
 
 func (e *ExecutorImpl) Verify(ctx context.Context, logE *logrus.Entry, param *ParamVerify, signature string) error {
+	if e == nil {
+		return errors.New("executor is nil")
+	}
 	// minisign -Vm myfile.txt -P RWQf6LRCGA9i53mlYecO4IzT51TGPpvWucNSCh1CBM0QTaLn73Y7GFO3
 	args := []string{
 		"-Vm",

--- a/pkg/minisign/package.go
+++ b/pkg/minisign/package.go
@@ -4,97 +4,100 @@ import (
 	"github.com/aquaproj/aqua/v2/pkg/config"
 	"github.com/aquaproj/aqua/v2/pkg/config/aqua"
 	"github.com/aquaproj/aqua/v2/pkg/config/registry"
-	"github.com/aquaproj/aqua/v2/pkg/ptr"
 )
 
 func Package() *config.Package { //nolint:funlen
-	return &config.Package{
-		Package: &aqua.Package{
-			Name:    "jedisct1/minisign",
-			Version: Version,
-		},
-		PackageInfo: &registry.PackageInfo{
-			Type:               "github_release",
-			RepoOwner:          "jedisct1",
-			RepoName:           "minisign",
-			VersionConstraints: `false`,
-			VersionOverrides: []*registry.VersionOverride{
-				{
-					VersionConstraints:  `Version == "0.11"`,
-					Asset:               "minisign-{{.Version}}-{{.OS}}.{{.Format}}",
-					Format:              "zip",
-					Rosetta2:            ptr.Bool(true),
-					WindowsARMEmulation: ptr.Bool(true),
-					Replacements: map[string]string{
-						"darwin":  "macos",
-						"windows": "win64",
-						"amd64":   "x86_64",
-						"arm64":   "aarch64",
-					},
-					Overrides: []*registry.Override{
-						{
-							GOOS:   "linux",
-							Format: "tar.gz",
-							Files: []*registry.File{
-								{
-									Name: "minisign",
-									Src:  "minisign-{{.OS}}/{{.Arch}}/minisign",
-								},
-							},
-						},
-						{
-							GOOS: "windows",
-							Files: []*registry.File{
-								{
-									Name: "minisign",
-									Src:  "minisign-win64/minisign.exe",
-								},
+	pkg := &aqua.Package{
+		Name:    "jedisct1/minisign",
+		Version: Version,
+	}
+	if Version == "0.11" {
+		return &config.Package{
+			Package: pkg,
+			PackageInfo: &registry.PackageInfo{
+				Type:                "github_release",
+				RepoOwner:           "jedisct1",
+				RepoName:            "minisign",
+				VersionConstraints:  `false`,
+				Asset:               "minisign-{{.Version}}-{{.OS}}.{{.Format}}",
+				Format:              "zip",
+				Rosetta2:            true,
+				WindowsARMEmulation: true,
+				Replacements: map[string]string{
+					"darwin":  "macos",
+					"windows": "win64",
+					"amd64":   "x86_64",
+					"arm64":   "aarch64",
+				},
+				Overrides: []*registry.Override{
+					{
+						GOOS:   "linux",
+						Format: "tar.gz",
+						Files: []*registry.File{
+							{
+								Name: "minisign",
+								Src:  "minisign-{{.OS}}/{{.Arch}}/minisign",
 							},
 						},
 					},
-					SupportedEnvs: []string{
-						"darwin",
-						"windows",
-						"amd64",
+					{
+						GOOS: "windows",
+						Files: []*registry.File{
+							{
+								Name: "minisign",
+								Src:  "minisign-win64/minisign.exe",
+							},
+						},
 					},
 				},
+				SupportedEnvs: []string{
+					"darwin",
+					"windows",
+					"amd64",
+				},
+			},
+		}
+	}
+	return &config.Package{
+		Package: pkg,
+		PackageInfo: &registry.PackageInfo{
+			Type:                "github_release",
+			RepoOwner:           "jedisct1",
+			RepoName:            "minisign",
+			VersionConstraints:  `false`,
+			Asset:               "minisign-{{.Version}}-{{.OS}}.{{.Format}}",
+			Format:              "zip",
+			WindowsARMEmulation: true,
+			Files: []*registry.File{
 				{
-					VersionConstraints:  `true`,
-					Asset:               "minisign-{{.Version}}-{{.OS}}.{{.Format}}",
-					Format:              "zip",
-					WindowsARMEmulation: ptr.Bool(true),
+					Name: "minisign",
+					Src:  "minisign-{{.OS}}/{{.Arch}}/minisign",
+				},
+			},
+			Replacements: map[string]string{
+				"darwin":  "macos",
+				"windows": "win64",
+				"amd64":   "x86_64",
+				"arm64":   "aarch64",
+			},
+			Overrides: []*registry.Override{
+				{
+					GOOS:   "linux",
+					Format: "tar.gz",
+				},
+				{
+					GOOS: "darwin",
 					Files: []*registry.File{
 						{
 							Name: "minisign",
-							Src:  "minisign-{{.OS}}/{{.Arch}}/minisign",
 						},
-					},
-					Replacements: map[string]string{
-						"darwin":  "macos",
-						"windows": "win64",
-						"amd64":   "x86_64",
-						"arm64":   "aarch64",
-					},
-					Overrides: []*registry.Override{
-						{
-							GOOS:   "linux",
-							Format: "tar.gz",
-						},
-						{
-							GOOS: "darwin",
-							Files: []*registry.File{
-								{
-									Name: "minisign",
-								},
-							},
-						},
-					},
-					SupportedEnvs: []string{
-						"darwin/arm64",
-						"windows",
-						"linux/amd64",
 					},
 				},
+			},
+			SupportedEnvs: []string{
+				"darwin/arm64",
+				"windows",
+				"linux/amd64",
 			},
 		},
 	}

